financial-connections/src/main/java/com/stripe/android/financialconnections/FinancialConnectionsSheetViewModel.kt

```diff
@@ -188,6 +189,10 @@ internal class FinancialConnectionsSheetViewModel @Inject constructor(
             linkMode?.let { queryParams.add("link_mode=${it.value}") }
         }
 
+        email?.let { queryParams.add("email=$it") }
+        phone?.let { queryParams.add("linkMobilePhone=$it") }
+        phoneCountry?.let { queryParams.add("linkMobilePhoneCountry=$it") }
```

Comment:
Doing this outside of Instant Debits, as this also applies to the Networking flow.